drivers/serial/leuart_gecko.c

```diff
@@ -10,10 +10,16 @@
 #include <zephyr/drivers/uart.h>
 #include <zephyr/irq.h>
 #include <em_leuart.h>
-#include <em_gpio.h>
+
 #include <em_cmu.h>
 #include <soc.h>
 
+#ifdef CONFIG_PINCTRL
+#include <zephyr/drivers/pinctrl.h>
+#else
+#include <em_gpio.h>
+#endif /* CONFIG_PINCTRL */
```

Comment:
please remove conditional pinctrl, there's no other mechanism today to configure pins.